presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraPageSink.java

```diff
@@ -177,6 +180,12 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             values.add(type.getSlice(block, position).toByteBuffer());
         }
+        else if (UuidType.UUID.equals(type)) {
+            Slice slice = type.getSlice(block, position);
```

Comment:
nit: can you extract helper method with this and put it in `io.prestosql.spi.type.UUIDs`. It follows the pattern we have e.g for `Decimals` and `Timestamps`.